### DISCUSSION TITLE
Multiple Issues with Deploying React to GitHub Pages

### DISCUSSION THREAD

**Discussion by WRNoble** [community/community#22738] [Other Feature Feedback, Questions, & Ideas]
<p>Good Morning, I’ve been attempting to deploy several React Apps to gh pages recently.  They all work fine on the local host.  I go through the standard deploy procedures and check my site only to find it blank the first time.  It had the following errors logged to the console. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/eed5e0a3dca4c793510c42c0c5bdafca12b2ce52.png" data-download-href="/uploads/short-url/y4PRQ7yGOxJMh1wiGiGdcHXVV1E.png?dl=1" title="Screen Shot 2020-08-03 at 9.45.12 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/59980165/181072765-b1af990e-5182-4c3a-8e87-914658b85749.png" alt="Screen Shot 2020-08-03 at 9.45.12 AM" data-base62-sha1="y4PRQ7yGOxJMh1wiGiGdcHXVV1E" width="690" height="62" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/eed5e0a3dca4c793510c42c0c5bdafca12b2ce52_2_690x62.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/eed5e0a3dca4c793510c42c0c5bdafca12b2ce52_2_1035x93.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/eed5e0a3dca4c793510c42c0c5bdafca12b2ce52_2_1380x124.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/eed5e0a3dca4c793510c42c0c5bdafca12b2ce52_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-08-03 at 9.45.12 AM</span><span class="informations">1438×131 31.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The second time the link loaded the readme, which is really unusual because there is no readme in the build directory.</p>
<p>Here is my package.json code</p>
<pre><code class="lang-auto">{
  "homepage": "https://wrnoble.github.io/gif-search-test",
  "name": "gif-search-test",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.5.0",
    "@testing-library/user-event": "^7.2.1",
    "axios": "^0.19.2",
    "gh-pages": "^3.1.0",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-router": "^5.2.0",
    "react-scripts": "3.4.1"
  },
  "scripts": {
    "predeploy": "npm run build",
    "deploy": "gh-pages -d build",
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
</code></pre>
<p>Any help would be greatly appreciated, I’ve stuck on this for a couple of days now.</p>

**Comment by WRNoble** [comment, marked as answer]
<p>Issue has been resolved!</p>

**Comment by muescha** [comment]
<p>Can you also share how you solved this issues?</p>
